prov/gni/include/fi_ext_gni.h

```diff
@@ -54,6 +54,8 @@ typedef enum dom_ops_val { GNI_MSG_RENDEZVOUS_THRESHOLD,
 			   GNI_MBOX_MSG_MAX_SIZE,
 			   GNI_RX_CQ_SIZE,
 			   GNI_TX_CQ_SIZE,
+			   GNI_MAX_RETRANSMITS,
```

Comment:
This should be added to the man page.  Also, will the MSG operations use the same value?